playbooks/group_vars/all/public.yml

```diff
@@ -7,7 +7,7 @@ ansible_python_interpreter: /usr/bin/python3
 # TARSNAP #####################################################################
 
 tarsnap_version: 1.0.39
-tarsnap_cache: /var/tarsnap/cache
+tarsnap_cache_dir: /var/tarsnap/cache
```

Comment:
@itsjeyd : Here is my [reason](https://tasks.opencraft.com/browse/OC-4845?focusedCommentId=78927&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-78927) for having two variable namely `tarsnap_cache_dir` and `TARSNAP_CACHE`. Please correct me if I am proceeding in the wrong direction.

Reply:
@taranjeet Thanks, I had seen that comment :) But I'm not sure which part of it explains why we need to rename `tarsnap_cache` to `tarsnap_cache_dir` (and can't just keep `tarsnap_cache` and `TARSNAP_CACHE`)?

It seemed to me like you were mainly analyzing what the value of `tarsnap_cache` ultimately gets set to for the different invocations of the `tarsnap` role listed [here](https://github.com/open-craft/ansible-playbooks/blob/master/playbooks/roles/backup-swift-container/meta/main.yml). Could you please clarify?

Reply:
@itsjeyd : I thought `tarsnap_cache` is an internal variable, and to avoid confusing, renaming it will be a better option. That way, one can easily get that `tarsnap_cache_dir` is an internal variable and whenever one needs to override that value, one can simply use `TARSNAP_CACHE` (like its happening at several places).

I will update this if this is not the right approach.